### PR TITLE
admin QoL changes

### PR DESCRIPTION
**Pull request by infraredbaron**
## About The Pull Request
Adds logs to certain events, such as: sanity breakdowns, all offence litanies, baptizing, successful roleset, contract competition, shuttle moving and _RnD research wiping._
+ Grab logs for attack log, assailant and victim

### CODE REVIEW

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -308,6 +308,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		griefProtection()
 		var/choice = alert("R&D Console Database Reset", "Are you sure you want to reset the R&D console's database? Data lost cannot be recovered.", "Continue", "Cancel")
 		if(choice == "Continue")
+			log_and_message_admins("reset the R&D console's database")
```
Comment (TorinoFermic) [inline code/modules/research/rdconsole.dm]: the var usr is accessible if you want know who did it

Comment (infraredbaron) [reply code/modules/research/rdconsole.dm]: it already does it, for some reason, so I decided to get rid of it

Comment (TorinoFermic) [reply code/modules/research/rdconsole.dm]: Looks fine then in the code itself.

**Comment by drexample** [comment]
@infraredbaron rebase your branch pls

**Comment by drexample** [comment]
@infraredbaron rebase your branch and fix the conflicts

**Comment by infraredbaron** [comment]
I'll do that after one last addition

**Pull request merged**